src/containers/CreatePipelineRun/CreatePipelineRun.js

```diff
@@ -14,497 +14,427 @@ limitations under the License.
   }
 
-  checkFormValidation() {
-    const reducer = (acc, key) => {
-      if (key === 'namespace') {
-        return this.checkValidation(key, this.getNamespace()) && acc;
-      }
-      if (key === 'pipeline') {
-        return this.checkValidation(key, this.getPipeline()) && acc;
-      }
-      return this.checkValidation(key, this.state[key].value) && acc;
-    };
-    return Object.keys(formValidation).reduce(reducer, true);
+  resetParamsState() {
+    this.setState((state, props) => ({
+      params: initialParamsState(
+        this.getPipelineInfo(PARAM_SPECS, { state, props })
+      )
+    }));
   }
 
-  reset() {
-    this.setState(initialState);
+  resetResourcesState() {
+    this.setState((state, props) => ({
+      resources: initialResourcesState(this.getPipelineInfo(RESOURCE_SPECS), {
+        state,
+        props
+      })
+    }));
   }
 
   render() {
-    const { pipelineName, open, namespace } = this.props;
-    const { errorMessage } = this.state;
-    const namespaceValue = this.getNamespace();
-    const pipelineValue = this.getPipeline();
-
+    const { open } = this.props;
+    const { serviceAccount, validationError } = this.state;
+    const namespace = this.getPipelineInfo(NAMESPACE);
+    const pipelineRef = this.getPipelineInfo(PIPELINE_REF);
+    const pipelineInfoDisabled = this.getPipelineInfo(DISABLED);
+    const resourceSpecs = this.getPipelineInfo(RESOURCE_SPECS);
+    const paramSpecs = this.getPipelineInfo(PARAM_SPECS);
     return (
-      <Form onSubmit={this.onSubmit}>
-        <ComposedModal
+      <Form>
+        <Modal
           className="create-pipelinerun"
-          onClose={this.onClose}
           open={open}
+          modalHeading="Create PipelineRun"
+          modalLabel={this.getPipelineInfo(PIPELINE_REF)}
+          primaryButtonText="Submit"
+          secondaryButtonText="Cancel"
+          onRequestSubmit={this.handleSubmit}
+          onRequestClose={this.handleClose}
+          onSecondarySubmit={this.handleClose}
         >
-          <ModalHeader
-            id="create-pipelinerun--header"
-            title="Create PipelineRun"
-            label={pipelineName}
-            closeModal={this.onClose}
-          />
-          <ModalBody>
-            {errorMessage !== '' && (
-              <InlineNotification
-                kind="error"
-                hideCloseButton
-                lowContrast
-                title="Error creating PipelineRun"
-                subtitle={errorMessage}
-              />
-            )}
-            {this.state.submitValidationError && (
-              <InlineNotification
-                kind="error"
-                hideCloseButton
-                lowContrast
-                title="Unable to submit"
-                subtitle="Please fix the fields with errors"
-              />
-            )}
+          {this.getPipelineInfo(PIPELINE_ERROR) && (
+            <InlineNotification
+              kind="error"
```

Comment:
notifications may require the `lowContrast` prop to restore previous styling since the Carbon 10.3 update (switched to a high contrast dark theme by default for some components)

also shouldn't be necessary to specify the `subtitle` if it's being left blank